examples/angular/src/pages/ui/primitive-poc/text-field/text-field.component.html

```diff
@@ -0,0 +1,44 @@
+<amplify-poc-text-field
+  labelHidden="true"
+  describedBy="custom-aria-describedBy"
+>
+  <p amplify-label>Last name</p>
+  <p amplify-description-text>Enter Description</p>
+  <input amplify-input placeholder="Baggins" />
+  <p amplify-error>There is an error</p>
+</amplify-poc-text-field>
+<br />
+<br />
+<br />
+<br />
```

Comment:
NIT: these `<br />`s are getting repetitive, we can think about using CSS or some other abstractions to handle these.

Reply:
Done.